ajax/dashboard.php

```diff
@@ -108,6 +109,21 @@
       echo $grid->getCardHtml($_REQUEST['card_id'], $_REQUEST);
       break;
 
+   case 'get_cards':
+      session_write_close();
+      header("Content-Type: application/json; charset=UTF-8");
+      // Parse stringified JSON payload (Used to preserve integers)
+      $request_data = array_merge($_REQUEST, json_decode(html_entity_decode(stripslashes($_REQUEST['data'])), true));
```

Comment:
You can use the non sanitized request, I guess.
```suggestion
      $request_data = array_merge($_REQUEST, json_decode($_UREQUEST['data'], true));
```